docs/examples/Parameters/Parameters.ipynb

```diff
@@ -6,72 +6,108 @@
    "source": [
     "# Parameters in QCoDeS\n",
     "\n",
-    "A `Parameter` is the basis of measurements and control within QCoDeS. Anything that you want to either measure or control within QCoDeS should satisfy the `Parameter` interface. You may read more about the `Parameter` [here](http://qcodes.github.io/Qcodes/user/intro.html#parameter).\n"
+    ">need to talk about get and set\n",
+    "\n",
+    "A `Parameter` is the basis of measurements and control within QCoDeS. Anything that you want to either measure or control within QCoDeS goes thought the `Parameter` interface as it represents the state variables of the system. While many Parameters represent a setting or measurement for a particular `Instrument`, it is possible to define `Parameters` that represent more powerful abstractions.\n",
```

Comment:
Check that instrument, parameter etc sections are elsewhere and remove from here